src/main/java/teammates/ui/webapi/CreateFeedbackSessionLogAction.java

```diff
@@ -0,0 +1,52 @@
+package teammates.ui.webapi;
+
+// import org.apache.http.HttpStatus;
```

Comment:
Remove and the TODO below? Place the code into another PR for the actual logic

Reply:
Done